ansible_base/resource_registry/views.py

```diff
@@ -131,7 +131,17 @@ def manifest(self, request, name, *args, **kwargs):
         if not resource_type.serializer_class:  # pragma: no cover
             return HttpResponseNotFound()
 
-        resources = Resource.objects.filter(content_type__resource_type=resource_type).prefetch_related("content_object")
+        if 'service_id' in request.query_params:
+            if request.query_params['service_id']:
+                service_filter = {'service_id': request.query_params['service_id']}
+            else:
+                service_filter = {}
```

Comment:
IMO `?service_id=` is a bit strange, looks like a request error.

I would prefer explicitly `?service_id=all`

Reply:
ok, added that. Could use docs, but this should only matter internally for now.